sys/include/net/ieee802154/radio.h

```diff
@@ -0,0 +1,1086 @@
+/*
+ * Copyright (C) 2020 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    drivers_ieee802154_hal IEEE802.15.4 Radio Hardware Abstraction Layer
+ * @ingroup     drivers
+ * @brief       This is a Hardware Abstraction Layer for IEEE802.15.4 compatible
+ *              radios.
+ *
+ * @author      José I. Alamos <jose.alamos@haw-hamburg.de>
+ */
+
+#ifndef IEEE802154_RADIO_H
+#define IEEE802154_RADIO_H
+
+#include <stdbool.h>
+#include "iolist.h"
+#include "sys/uio.h"
+#include "byteorder.h"
+#include "net/eui64.h"
+
+/**
+ * @brief Forward declaration of the radio ops structure.
+ */
+typedef struct ieee802154_radio_ops ieee802154_radio_ops_t;
+
+typedef enum {
+    /**
+     * @brief the transceiver received a valid ACK with the frame pending bit
+     */
+    TX_FLAGS_FRAME_PENDING=(1<<0),
```

Comment:
See [comment](https://github.com/RIOT-OS/RIOT/pull/14371/files#r446917486)

Reply:
I cannot go back to that comment for some reason. Do you remember what this was about?

Reply:
Missing whitespace I guess. Should be fixed already by uncrustify

Reply:
marked as resolved then